File: dataset/human/python/file_2868.py
def fanction(n):
    listIn = []
    tmp = n
    for i in range(2, int(-(-n ** 0.5 // 1)) + 1):
        if tmp % i == 0:
            cnt_in = 0
            while tmp % i ==0:
                cnt_in += 1
                tmp //= i
            listIn.append([i, cnt_in])
 
    if tmp != 1:
        listIn.append([tmp, 1])
 
    return listIn
    
N = int(input())
 
arr = fanction(N)
 
tmp = 1
count = 0
for a, b in arr:
    while 'true':
        if b >= tmp:
            b -= tmp
            count += 1
            tmp += 1
        else:
            tmp = 1
            break
 
print(count)

File: dataset/human/c/file_2868.c
#include <stdio.h>

int main(void)
{
	int k, s;
	int x, y, z;
	int count = 0;
	
	scanf("%d %d", &k, &s);
	
	for(x=0; x<=k; x++){
		if( (x+2*k) < s )
			continue;
		else if(x > s)
			break;
		
		for(y=0; y<=k; y++){
			if( (x+y+k) < s )
				continue;
			else if( (x+y) > s )
				break;
			else if( s-(x+y) >= 0 )
				count++;
			//for(z=0; z<=k; z++)
		}
	}
	
	printf("%d\n", count);
	
	
	return 0;
}
